pegasus/sites.v3/code.org/public/private/poste/index.haml

```diff
@@ -83,7 +83,8 @@
     Showing most recently sent mail. Click to view all mail.
 
 :ruby
-  rows = DB.fetch("
+  POSTE_REPORTING_DB_READONLY = sequel_connect(CDO.pegasus_reporting_db_reader, CDO.pegasus_reporting_db_reader)
```

Comment:
Not in any common location, it is defined in random files.

For now, since this page is going to start getting some heavy traffic, I'm going to merge this PR and worry about making things pretty in a later PR.